mage_ai/server/data/base.py

```diff
@@ -46,30 +46,43 @@ def read_parquet_file(self, file_name):
         file_path = os.path.join(self.dir, file_name)
         if not os.path.exists(file_path):
             return pd.DataFrame()
-        return pd.read_parquet(file_path, engine='pyarrow',)
+        return pd.read_parquet(file_path, engine='pyarrow')
 
     def write_parquet_file(self, file_name, df):
         df.to_parquet(os.path.join(self.dir, file_name))
 
     def to_dict(self, detailed):
         pass
-    
+
     @classmethod
     def folder_name(cls):
         return cls.__name__
 
+    @classmethod
+    def is_valid_id(cls, id):
+        opts = Model.gen_integer_dir_list(cls.path_name())
+        return int(id) in opts
+
     @classmethod
     def path_name(cls):
         return os.path.join(DATA_PATH, cls.folder_name())
 
     @classmethod
     def objects(cls):
         arr = []
-        dirs = sorted([int(name) for name in os.listdir(cls.path_name()) if name.isnumeric()],
-                      reverse=True)
-        for id in dirs:
+        for id in Model.gen_integer_dir_list(cls.path_name()):
```

Comment:
@skunichetty could you add the sorted function back?

Reply:
Oops, not sure how that got deleted. I'll add that back in right now.